ecs-cli/modules/version/version.go

```diff
@@ -17,15 +17,13 @@
 // ecs-cli. It is autogenerated
 package version
 
-// Please DO NOT commit any changes to this file (specifically the hash) except
-// for those created by running ./scripts/update-version at the root of the
-// repository. Only the 'Version' const should change in checked-in source code
+// Please DO NOT commit any changes to this file.
 
-// Version is the version of the ECS CLI
-const Version = "1.15.1"
```

Comment:
Removing this and `version-gen.go` break the `ecs-cli --version` cmd when building the binary from source:

```
// test on Windows, but same results seen using MAKE on linux

> ./bin/local/ecs-cli.exe --version
ecs-cli version  ()
```

We should retain this functionality when building outside of CodeBuild. Is there a way to move the tag/version retrieval logic from the buildspec.yml into the code somewhere so it can be reused when building locally?